### DISCUSSION TITLE
Change errors path to project root path

### DISCUSSION THREAD

**Discussion by DraftProducts** [eslint/eslint#15115] [Q&A] [answered]
As you know when you start a project in docker container, the path of errors is based on the root path:
![image](https://user-images.githubusercontent.com/29425008/135086950-002f39bd-4a01-4829-9e01-54e07dcbfad4.png)

Is there a way to add an option to ESLint to start on the project root path ?

**Comment by nzakas** [comment, marked as answer]
ESLint uses the current working directory as it’s based. There’s no way to change that.

**Reply by DraftProducts** [reply]
Ok thanks 👍
